src/main/java/baseball/controller/GameController.java

```diff
@@ -0,0 +1,58 @@
+package baseball.controller;
+
+import baseball.domain.Computer;
+import baseball.domain.Referee;
+import baseball.domain.Result;
+import baseball.util.GameProgress;
+import baseball.view.InputView;
+import baseball.view.OutputView;
+import java.util.List;
+
+public class GameController {
+
+    private final Computer computer;
+    private final Referee referee;
+    private GameProgress gameProgress;
+
+    public GameController() {
+        this.computer = new Computer();
+        this.referee = new Referee();
+        this.gameProgress = GameProgress.START;
+    }
+
+    public void play() {
+        playOrEnd();
+    }
+
+    private void playOrEnd() {
+        OutputView.printStart();
+        while (!GameProgress.isQuit(gameProgress)) {
```

Comment:
while의 조건문을 긍정문으로 하는 것이 가독성에 더 도움이 된다고 하더라구요. 
```suggestion
        while (GameProgress.isRetry(gameProgress)) {
```
이렇게 바꿔보시는 것도 한번 제안드려봅니다!